README.md

```diff
@@ -564,6 +564,17 @@ If you change something in HTML/CSS/JS and want to create a pull request, please
 do include the minified versions and corresponding C header files in your
 commits. (Do **NOT** commit all the minified versions for the non changed files)
 
+# Experimental note for Development:
```

Comment:
Could this be a more descriptive title? 

"Debugging using Host Emulation ", or similar?

Reply:
Title is changed to 
> Experimental debugging environment using emulation on host